Packs/TaniumThreatResponse/Integrations/TaniumThreatResponse/TaniumThreatResponse.py

```diff
@@ -14,6 +14,7 @@
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
 ''' GLOBALS/PARAMS '''
+REQUEST_TIMEOUT = 60
```

Comment:
60 seconds is a lot. The timeout is 10s? Let's try it with 20s, it could be very painful to customers.